mmv1/templates/terraform/examples/node_group_share_settings.tf.erb

```diff
@@ -0,0 +1,30 @@
+resource "google_project" "guest_project" {
+  project_id      = "tf-test-%{random_suffix}"
+  name            = "tf-test-%{random_suffix}"
+  org_id          = "<%= ctx[:test_env_vars]['org_id'] %>"
+}
+
+resource "google_compute_node_template" "soletenant-tmpl" {
+  project = google_project.owner_project.project_id
```

Comment:
This line can be removed as we don't need specify project. The default project from the provider will be used.

Reply:
Done, thank you!